### DISCUSSION TITLE
Consider switching to Axum?

### DISCUSSION THREAD

**Discussion by niulai6** [sparckles/Robyn#705] [General]
https://crates.io/crates/axum

```
Downloads all time
26,178,149
```

https://crates.io/crates/actix

```
Downloads all time
6,254,873
```

**Comment by sansyrox** [comment]
Hey @xin-hao-awx 👋 

Thank you for your suggestion. But Actix is still faster than axum in most cases (https://www.techempower.com/benchmarks/#section=data-r21&test=composite&hw=ph) . Also, this migration will not benefit us as much as the pizzaz of axum is in its ergonomics. However, that is taken care of by us internally.
